client/lib/posts/test/post-list-store.js

```diff
@@ -0,0 +1,241 @@
+/* eslint-disable vars-on-top */
+require( 'lib/react-test-env-setup' )();
```

Comment:
Since these tests don't use the DOM, there's no need to include `lib/react-test-env-setup`